### PR TITLE
GA-01

### PR DESCRIPTION
**Pull request by TheCsarbeat**
# Description

Explored the dataset to gain more insight of the features and the values. Noticed that there were several `0` values in the rows which were taken for `null` values upon closer investigation. Then replaced said values for the `mean` of that specific column. Except for columns _Pregnancies_ and _Outcome_.

# Related issues

Task: #1 

# Screenshots

image
image
image
image
image
image

# Additional notes

- Having replaced the `0` values for the `mean` is one possible approach, however if there is any other better alternative for this specific dataset, you're welcome to make any suggestions.
- As seen in the graphs, there is way more samples of people between the ages of 20 and 30
- The amount samples of non-diabetic people are greater than diabetic people
- There seems to be that diabetic people tend to have thicker skin as well as higher levels of glucose, however, these differences among groups (Diabetic, Non-Diabetic) are small.

### CODE REVIEW

File: p1.ipynb [hunk 1]
```diff
@@ -35,7 +35,7 @@
   },
```
Comment (TheCsarbeat) [inline p1.ipynb:35]: <p>prueba de comentario, qué buen histograma, usted es un genio la verdad</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TheCsarbeat/proyecto1-IA/pull/2/discussion/#comment-1545907146'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

Comment (fernandamurillo04) [reply p1.ipynb:35]: <p>hola</p>

Comment (Chacalerks) [reply p1.ipynb:35]: <p>Test of comment by ReviewNB</p>

**Review by Chacalerks — APPROVED** [review]
All its good, I suggest we should add text before each function and graphic this allow to create the paper easily

**Review by fernandamurillo04 — APPROVED** [review]

**Pull request merged**